app/src/main/res/values/strings.xml

```diff
@@ -84,11 +84,13 @@
     <string name="contact_image">Contact image</string>
 
     <!-- Select verification method -->
-    <string name="select_verification_method">Select a verification method</string>
+    <string name="select_verification_method">How do you want to pair?</string>
     <string name="select_visual_verification">Visual pattern</string>
     <string name="select_rhythm_verification">Tap a rhythm</string>
     <string name="unknown_verification_method_title">Oops…</string>
     <string name="unknown_verification_method_description">Something went wrong, please try again</string>
+    <string name="by_tapping_a_rhythm">By tapping a rhythm</string>
+    <string name="by_tapping_on_an_image">By tapping on an image</string>
```

Comment:
I don't think this is the correct way to put it, maybe: "By selecting a few things in an image"?